direct/nn/unet/config.py

```diff
@@ -31,3 +31,12 @@ class Unet2dConfig(ModelConfig):
     skip_connection: bool = False
     normalized: bool = False
     image_initialization: str = "zero_filled"
+
```

Comment:
I know this is old, but can't we change this to Enum's?

Reply:
Also remove the first line while you're at it.

Reply:
Done!